dxr/templates/nunjucks/path_line.html

```diff
@@ -1,6 +1,4 @@
 {%- if not is_first_or_only -%}
 <span class="path-separator">/</span>
 {%- endif -%}
-<a href="{{ url }}" {%- if is_dir %} data-path="{{ data_path }}" {% endif %}>
-    {{ display_path }}
-</a>
+<a href="{{ url }}" {%- if is_dir %} data-path="{{ data_path }}" {% endif %}>{{ display_path }}</a>
```

Comment:
This is fine. And in case you didn't know, you can say `{{- ... -}}` (IIRC) to trim surrounding whitespace from `{{ ... }}` expressions.